snap7/client/__init__.py

```diff
@@ -1000,7 +1004,7 @@ def wait_as_completion(self, timeout: int) -> int:
         check_error(result, context="client")
         return result
 
-    def _prepare_as_read_area(self, area: Areas, size: int) -> Tuple[WordLen, Array]:
+    def _prepare_as_read_area(self, area: Areas, size: int) -> Tuple[WordLen, "Array[_SimpleCData[int]]"]:
```

Comment:
why does this type need to be a a string?

Reply:
https://github.com/gijzelaerr/python-snap7/pull/516#discussion_r1623561951